### ISSUE TITLE
Non-existing config file for Job Declarator client does not result in error

### ISSUE THREAD

**Issue by Sjors** [stratum-mining/stratum#695]
```
cd roles/jd-client
cargo run -- -c ~/no-such-file.toml
warning: profiles for the non root package will be ignored, specify profiles at the workspace root:
package:   /home/sjors/dev/stratum/test/scale/Cargo.toml
workspace: /home/sjors/dev/stratum/Cargo.toml
warning: /home/sjors/dev/stratum/Cargo.toml: unused manifest key: authors
warning: /home/sjors/dev/stratum/Cargo.toml: unused manifest key: description
warning: /home/sjors/dev/stratum/Cargo.toml: unused manifest key: documentation
warning: /home/sjors/dev/stratum/Cargo.toml: unused manifest key: edition
warning: /home/sjors/dev/stratum/Cargo.toml: unused manifest key: homepage
warning: /home/sjors/dev/stratum/Cargo.toml: unused manifest key: keywords
warning: /home/sjors/dev/stratum/Cargo.toml: unused manifest key: license
warning: /home/sjors/dev/stratum/Cargo.toml: unused manifest key: license-file
warning: /home/sjors/dev/stratum/Cargo.toml: unused manifest key: name
warning: /home/sjors/dev/stratum/Cargo.toml: unused manifest key: readme
warning: /home/sjors/dev/stratum/Cargo.toml: unused manifest key: repository
warning: /home/sjors/dev/stratum/Cargo.toml: unused manifest key: rust-version
warning: /home/sjors/dev/stratum/Cargo.toml: unused manifest key: version
    Finished dev [optimized + debuginfo] target(s) in 0.06s
     Running `/home/sjors/dev/stratum/target/debug/jd_client -c /home/sjors/no-such-file`
```

This should throw an error that the file doesn't exist.

Similarly I suspect, but still debugging, that it doesn't report errors in the config and instead just exits.

**Comment by Sjors** [comment]
The translator also doesn't care if I point to a config file that doesn't exist.

**Comment by plebhash** [comment]
the fix here should be relatively simple, so I'll label as `good first issue`

also, it should wait until https://github.com/stratum-mining/stratum/pull/814 is merged in order to apply a standardized solution to all roles

**Comment by rockcoolsaint** [comment]
Can I give this issue a go as it seems no one is assigned to it yet? @plebhash

**Comment by plebhash** [comment]
> Can I give this issue a go as it seems no one is assigned to it yet? @plebhash 

sure, but I think it's better that you start from the branch I've been working on #814

otherwise we would be proposing conflicting CLI code

**Comment by plebhash** [comment]
`translator` role suffers from this same bug

**Comment by plebhash** [comment]
closed via #859
